packages/amplify-graphql-api-construct-tests/src/__tests__/backends/sql-models/app.ts

```diff
@@ -28,12 +28,21 @@ interface DBDetails {
     };
   };
   dbConnectionConfig: SqlModelDataSourceDbConnectionConfig;
+  schemaConfig: string | null;
```

Comment:
nit: Do you anticipate additional configuration options that will be added here or if not just saying `schema: string|undefined` is more intuitive.

Reply:
(Also, piling onto @phani-srikar 's comment: prefer `string | undefined` over `string | null` unless you actually mean "the value must be a literal `null`". `undefined` means "unassigned or not specified", which is usually what you want.)

Reply:
`Undefined` is used for `null` for more intuitive and precise "unassigned" value.